### ISSUE TITLE
Problems with my Git 

### ISSUE THREAD

**Issue by golnar-bt** [cfss-old/sp18_hw01#52]
I was trying to commit my changes for the first few questions I answered. However, nothing shows up in my git tab Rstudio! I did not have this issue before with HW1. 
![img_0700 jpg](https://user-images.githubusercontent.com/37814296/38379215-a3e829d0-38c5-11e8-9c70-88979e8914d7.jpeg)

**Comment by bensoltoff** [comment]
Have you actually saved the file you were editing?

**Comment by golnar-bt** [comment]
Yes I did.

**Comment by bensoltoff** [comment]
Are you sure the files you are editing are located in your `hw02` folder on your computer? Might you have accidentally saved them in a different location?

**Comment by golnar-bt** [comment]
Yes. I just deleted everything and restarted multiple times.I kept trying and trying and it just suddenly worked. I have no idea how and what I did differently ! But thank you !
